projects/plugins/jetpack/class.json-api-endpoints.php

```diff
@@ -277,6 +285,11 @@ abstract class WPCOM_JSON_API_Endpoint {
 	 */
 	public $allow_fallback_to_jetpack_blog_token = false;
 
+	/**
+	 * REST namespace.
+	 */
+	const REST_NAMESPACE = 'rest/v1';
```

Comment:
Let's omit the version part here since we're gonna need to dynamically calculate it based on each endpoint's `max_version` definition.